source/Timezones.mc

```diff
@@ -0,0 +1,114 @@
+using Toybox.WatchUi as Ui;
+using Toybox.System as Sys;
+using Toybox.Application as App;
+using Toybox.Graphics as Gfx;
+using Toybox.Time.Gregorian as Calendar;
+
+(:timezones)
+class Timezones extends Ui.Drawable {
+
+	var utcOffset = new Time.Duration(-Sys.getClockTime().timeZoneOffset);
+	
+	var tzLocations = [];
+	var UTC_Offsets = {};
+	var mTimeZones;
+	
+	var mhalfDCWidth; 
+	
+	var mTopY;
+	var mBottomY;
+
+    var faceWidth;
+    var faceHeight;
+
+	function initialize(params) {
+		Drawable.initialize(params);
+
+		mTopY = params[:topY];
+		mBottomY = params[:bottomY];
+		
+		setupTimezones();
+	}
+
+	function onSettingsChanged() {
+		setupTimezones();
```

Comment:
`setupTimezones()` could be combined with `onSettingsChanged()`, then `initialize()` could call `onSettingsChanged()`, saving a member function (see comment on `drawTimezones()`).

Reply:
`mTimeZones` could then be a local variable.